test/jdk/java/util/Random/RandomTestBsi1999.java

```diff
@@ -439,7 +439,10 @@ public static void main(String[] args) {
         RandomGeneratorFactory.all().forEach(factory -> {
             setRNG(factory.name());
 
-            if (factory.name().equals("Random")) {
+            if (currentRNG.equals("SecureRandom")) {
+                // Bug: 8264791
```

Comment:
It is generally not necessary to include bug ids in the text of changesets.

Reply:
Should you add an "@bug 8264791" in the header instead?